hemmingway-editor/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+archivedreason: 
+title: Do you use the Hemmingway Editor to improve your writing?
+guid: 2057dc3f-d424-4637-bfcd-3ef3809b5e75
+uri: hemmingway-editor
+created: 2023-10-16T05:06:33.0000000Z
+authors:
+- title: Jayden Alchin
+  url: https://ssw.com.au/people/jayden-alchin
+related:
+- concise-writing
+- use-active-voice
+redirects: []
+---
+
+The Hemmingway Editor is a tool that analyzes content and highlights problematic areas. 
+This process reveals common writing issues that impact readers. Lean on this tool to internalize good writing practices.
+
+<!--endintro-->
+  
+## Why use Hemmingway?
+
+This is a tool that analyzes text for readability. It's particularly useful for longer pieces, like articles or blog posts. 
+
+It highlights:
+- Use of passive voice 
+- Simple alternatives to complex phrases
+- Overuse of adverbs 
+- Long and hard to read sentences
+
+The editor also reports a grade-level readability score. Grade 9 reading level or below indicates the most understandable content.
+
+## How to use it
+
+Copy and paste content into the editor for immediate feedback. Action as many suggestions as possible, and try to achieve that grade 9 readability or better. 
+
+::: bad  
+![Figure: Bad example - Hemmingway editor highlights readability issues](hemmingway-bad.png)
```

Comment:
Hey @jaydenalchin 

I don't think this should be set as "bad example". In fact it's using the editor, so it's a good thing.

IMO this should be a normal image showing how Hemmingway works to improve your content. Then we have the good example of a clear, consice content...

Agree?

Reply:
I think the rule is about the content itself - so it is the bad example.

The editor is just the super easy means to the end

Reply:
I agree with Luke it's more about bad/good content and how the tool helps fix that. Made a small wordage change to make the example more about the content than the tool.